netbout-client/src/main/java/com/netbout/client/RtBoutIterator.java

```diff
@@ -53,6 +53,7 @@
 @ToString
 @Loggable(Loggable.DEBUG)
 @EqualsAndHashCode(of = { "request", "bouts" })
+@SuppressWarnings("PMD.ConstructorShouldDoInitialization")
```

Comment:
@danff I don't understand your suppression here... What's preventing `bouts` or `more` to be initialized in constructor?